mindmeld/models/nn_utils/layers.py

```diff
@@ -0,0 +1,269 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+from .._util import _get_module_or_attr
+
+try:
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+    from torch.nn.utils.rnn import pack_padded_sequence, pad_packed_sequence, pad_sequence
+
+    nn_module = _get_module_or_attr("torch.nn", "Module")
+except ImportError:
+    nn_module = object
+    pass
+
+SEED = 7246
+
+logger = logging.getLogger(__name__)
+
+
+# utils
+
+def get_disk_space_of_model(pytorch_module):
+    filename = "temp.bin"
+    torch.save(pytorch_module.state_dict(), filename)
```

Comment:
use sys.getsizeof instead of writing to disk

Reply:
replied in this comment <https://github.com/cisco/mindmeld/pull/325#discussion_r771109109>